src/webrtc/call.js

```diff
@@ -972,6 +973,11 @@ const setState = function(self, state) {
  * @return {Promise}
  */
 const sendEvent = function(self, eventType, content) {
+    // XXX: This makes this promise always succeed (ie. swallows the error)
```

Comment:
This comment is a bit hard to grok. Why is what the candidate-sending call does relevant here?

Reply:
Basically I was trying to explain why I added a catch to the send-candidate call even though it'll never get hit and that it's sort-of OK that this error gets swallowed. Any suggestions on how I can word this better?

Reply:
well, I suggest noting the fact that the send-candidate's `catch` is just for hygiene in the send-candidate code, and just losing the "but only one ... " phrase here.

Reply:
No :-p

"sending one sendEvent call"?